code/modules/mob/living/carbon/superior_animal/human/subtype/solelite/solelite

```diff
@@ -0,0 +1,100 @@
+/mob/living/carbon/superior_animal/human/solelite
+	name = "Elite Operative"
+	desc = "Black and charred hardsuit, sub-machine gun in-hand, a being of absolute terror to those who dare approach."
+	icon = 'icons/mob/mobs-humanoid.dmi'
+	icon_state = "syndicate_stormtrooper_smg"
+	stop_automated_movement_when_pulled = 1
+	wander = 0
+	maxHealth = 300
+	health = 300
+
+	armor = list(melee = 55, bullet = 55, energy = 70, bomb = 80, bio = 90, rad = 25) //Legitmently their armor (melee is higher {45} to account for AI stupidness)
+
+	//range/ammo stuff
+	ranged = 1
+	rapid = 1
+	rapid_fire_shooting_amount = 5
+	ranged_cooldown = 2
+	projectiletype = /obj/item/projectile/bullet/magnum_40/hv
+	projectilesound = 'sound/weapons/guns/fire/smg_fire.ogg'
+	limited_ammo = TRUE
+	mag_drop = TRUE
+	rounds_left = 20
+	mag_type = /obj/item/ammo_magazine/smg_magnum_40/empty
+	mags_left = 4 //4+1
+
+	melee_damage_lower = 20
+	melee_damage_upper = 30
+	breath_required_type = 0 // Doesn't need to breath, in a hardsuit
+	breath_poison_type = 0 // Can't be poisoned
+	min_air_pressure = 0 // Doesn't need pressure
+	attacktext = "slashed"
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	meat_amount = 0
+	meat_type = null
+	leather_amount = 0
+	bones_amount = 0
+//They are all wearing suits
+	breath_required_type = NONE
+	breath_poison_type = NONE
+	min_breath_required_type = 0
+	min_breath_poison_type = 0
+
+	min_air_pressure = 0
+	min_bodytemperature = 0
+
+//Drops
+	meat_amount = 1
+	meat_type = /obj/item/reagent_containers/food/snacks/meat/human
+
+	inherent_mutations = list(MUTATION_HEART, MUTATION_LUNG, MUTATION_LIVER, MUTATION_BLOOD_VESSEL, MUTATION_MUSCLES, MUTATION_NERVES)
+
+	faction = "elitemerc"
+
+/mob/living/carbon/superior_animal/human/solelite/handle_breath(datum/gas_mixture/breath) //we have are own air supplies
+	return
+
+/mob/living/carbon/superior_animal/human/solelite/handle_environment(var/datum/gas_mixture/environment) //are armor legit is a hardsuit
+	return
+
+/mob/living/carbon/superior_animal/human/solelite/start_pulling(var/atom/movable/AM)
```

Comment:
This is good, I'd also suggest adding a grab sanity check to prevent players from trying to CQC the dude in powered armor. If you need an example of that, look at the render and seekers.